sdcm/sct_events/database.py

```diff
@@ -335,10 +336,14 @@ def _end_event(event_type: Type[ScyllaDatabaseContinuousEvent], match: Match):
         begun_events = event_filter.get_filtered()
 
         if not begun_events:
-            raise ContinuousEventRegistryException("Did not find any events of type {event_type}"
-                                                   "with period type {period_type}."
-                                                   .format(event_type=event_type,
-                                                           period_type=EventPeriod.BEGIN.value))
+            TestFrameworkEvent(
+                source=event_type.__name__,
+                message="Did not find any events of type {event_type}"
+                        "with period type {period_type}.".format(event_type=event_type,
+                                                                 period_type=EventPeriod.BEGIN.value),
+                severity=Severity.ERROR
```

Comment:
I don't think it should have ERROR severity, WARNING at max. @bentsi , what do you think ?

Reply:
Actually that was decided with Bentsi. But let see what he say

Reply:
ok